OpenQA/Qemu/DriveDevice.pm

```diff
@@ -98,6 +106,7 @@ sub gen_cmdline ($self) {
         $self->_push_ifdef(\@params, 'bootindex=', $self->bootindex) if (!$path->id || $path->id eq 'path0');
         $self->_push_ifdef(\@params, 'serial=', $self->serial);
         $self->_push_ifdef(\@params, 'num_queues=', $self->num_queues) if (($self->num_queues // -1) != -1);
+        push(@params, 'logical_block_size=' . $self->sector_size . ',physical_block_size=' . $self->sector_size) if $self->sector_size;
```

Comment:
I'm not sure this is a good idea, but we can try, physcal block size and logical block size are two different things, although having the same block size seems to be supported:

- https://utcc.utoronto.ca/~cks/space/blog/tech/SCSIBlocksizesDiscovery
- https://www.ibm.com/docs/en/linux-on-systems?topic=prr-valid-physical-block-size-combinations-lvm-physical-volumes

```suggestion
        push(@params, 'logical_block_size=' . $self->sector_size . ',physical_block_size=' . $self->sector_size) if $self->sector_size;
```

Reply:
It is very common because it is usually faster - the question would be if we want to test systems where it's not configured the same.

Reply:
Similar as in https://github.com/os-autoinst/os-autoinst/pull/2516#discussion_r1692684214 . If you guys say we need two separate settings then we can do that.

I can't spot the actual diff in your suggestion. What do you propose to change there?

Reply:
I was only confirming matching the size is the common case. If we have no use case for specifying two values I'd say we can always add HDDLOGICALSIZE later